org.code.toboggan.network/META-INF/MANIFEST.MF

```diff
@@ -10,7 +10,7 @@ Require-Bundle: org.eclipse.core.runtime,
  org.code.toboggan.core;bundle-version="1.0.0",
  org.code.toboggan.clientcore;bundle-version="2.0.0",
  org.eclipse.core.resources,
- org.eclipse.wb.core.lib
+ org.eclipse.wb.core.lib;bundle-version="1.9.0"
```

Comment:
I'll rebase on top of master which should fix this. I was the one who pushed those changes from Ben's computer when I was setting it up for him.